# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../pixman/hotfix.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./pixman-1.pc.in.orig	2007-08-31 13:18:06.000000000 -0400
+++ ./pixman-1.pc.in	2007-08-31 13:18:17.000000000 -0400
@@ -7,5 +7,5 @@
 Description: The pixman library (version 1)
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}/pixman-1 @DEP_CFLAGS@
-Libs: -L${libdir} -lpixman @DEP_LIBS@
+Libs: -L${libdir} -lpixman-1 @DEP_LIBS@
 
